<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_FindID</class>
 <widget class="QDialog" name="F_FindID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Btn_Back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Find">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>307</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>80</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>tip:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="Tip_ID"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="Edit_ID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>76</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>input id:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
